gov/governance/fn_registry.gno

```diff
@@ -0,0 +1,193 @@
+			parseUint64(params[2]), // votingWeightSmoothingDuration
+			parseUint64(params[3]), // quorum
+			parseUint64(params[4]), // proposalCreationhold
+			parseUint64(params[5]), // executionDelay
+			parseUint64(params[6]), // executionWindow
+		)
+		return nil
+	})
+
+	// region: Pool Path
+	registry.Register(consts.POOL_PATH, "SetFeeProtocol", func(params []string) error {
+		if err := hasDesiredParams(params, 2); err != nil {
+			return err
+		}
+		pl.SetFeeProtocol(
+			uint8(parseUint64(params[0])), // feeProtocol0
+			uint8(parseUint64(params[1])), // feeProtocol1
+		)
+		return nil
+	})
+
+	registry.Register(consts.POOL_PATH, "SetPoolCreationFee", func(params []string) error {
+		if err := hasDesiredParams(params, 1); err != nil {
+			return err
+		}
+		pl.SetPoolCreationFee(parseUint64(params[0])) // fee
+		return nil
+	})
+
+	registry.Register(consts.POOL_PATH, "SetWithdrawalFee", func(params []string) error {
+		if err := hasDesiredParams(params, 1); err != nil {
+			return err
+		}
+		pl.SetWithdrawalFee(parseUint64(params[0])) // fee
+		return nil
+	})
+
+	// region: Protocol fee
+	registry.Register(consts.PROTOCOL_FEE_PATH, "SetDevOpsPct", func(params []string) error {
+		if err := hasDesiredParams(params, 1); err != nil {
+			return err
+		}
+		pf.SetDevOpsPct(parseUint64(params[0])) // pct
+		return nil
+	})
+
+	// region: Router
+	registry.Register(consts.ROUTER_PATH, "SetSwapFee", func(params []string) error {
+		if err := hasDesiredParams(params, 1); err != nil {
+			return err
+		}
+		rr.SetSwapFee(parseUint64(params[0])) // fee
+		return nil
+	})
+
+	// region: Staker
+	registry.Register(consts.STAKER_PATH, "SetDepositGnsAmount", func(params []string) error {
+		if err := hasDesiredParams(params, 1); err != nil {
+			return err
+		}
+		sr.SetDepositGnsAmount(parseUint64(params[0])) // amount
+		return nil
+	})
+
+	registry.Register(consts.STAKER_PATH, "SetPoolTier", func(params []string) error {
+		if err := hasDesiredParams(params, 2); err != nil {
+			return err
+		}
+		sr.SetPoolTier(
+			params[0],              // pool
+			parseUint64(params[1]), // tier
+		)
+		return nil
+	})
+
+	registry.Register(consts.STAKER_PATH, "ChangePoolTier", func(params []string) error {
+		if err := hasDesiredParams(params, 2); err != nil {
+			return err
+		}
+		sr.ChangePoolTier(
+			params[0],              // pool
+			parseUint64(params[1]), // tier
+		)
+		return nil
+	})
+
+	registry.Register(consts.STAKER_PATH, "RemovePoolTier", func(params []string) error {
+		if err := hasDesiredParams(params, 1); err != nil {
+			return err
+		}
+		sr.RemovePoolTier(params[0]) // pool
+		return nil
+	})
+
+	registry.Register(consts.STAKER_PATH, "SetUnstakingFee", func(params []string) error {
+		if err := hasDesiredParams(params, 1); err != nil {
+			return err
+		}
+		sr.SetUnstakingFee(parseUint64(params[0]))
+		return nil
+	})
+
+	registry.Register(consts.STAKER_PATH, "SetWarmUp", func(params []string) error {
```

Comment:
this one too

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/448/commits/89d5550f18a744da388d244352c15347c035de5b